Services/DataX.Utilities/DataX.Utilities.Blob/BlobHelper.cs

```diff
@@ -153,9 +153,9 @@ public static async Task<List<string>> GetLastModifiedBlobContentsInBlobPath(str
             CloudBlobClient cloudBlobClient = storageAccount.CreateCloudBlobClient();
             CloudBlobContainer container = cloudBlobClient.GetContainerReference(containerName);
 
-            var filteredBlobs = container.ListBlobsSegmentedAsync(prefix: prefix, useFlatBlobListing: true, blobListingDetails: BlobListingDetails.None, maxResults: null, currentToken: null, options: null, operationContext: null).Result.Results.OfType<CloudBlockBlob>()
+            var filteredBlobs = await Task.Run(() => container.ListBlobsSegmentedAsync(prefix: prefix, useFlatBlobListing: true, blobListingDetails: BlobListingDetails.None, maxResults: null, currentToken: null, options: null, operationContext: null).Result.Results.OfType<CloudBlockBlob>()
```

Comment:
Why this change? i.e invoking the listBlobs in a separate task?

Reply:
If there are a lot of blobs in the target folder, it could probably affect the performance. Having this would help. Any concern? I agree it can be non-async call though.

Reply:
update not to use task.run just use await.